sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilters.scala

```diff
@@ -168,6 +168,8 @@ private[sql] object OrcFilters extends OrcFiltersBase {
       toJavaDate(localDateToDays(value.asInstanceOf[LocalDate]))
     case _: TimestampType if value.isInstanceOf[Instant] =>
       toJavaTimestamp(instantToMicros(value.asInstanceOf[Instant]))
+    case _: TimestampNTZType =>
+      toJavaTimestamp(localDateTimeToMicros(value.asInstanceOf[LocalDateTime]))
```

Comment:
shall we use `OrcUtils.toOrcNTZ`?

Reply:
Hive not recognizes `OrcTimestamp`. Hive only have `Timestamp`.